portfolio/src/main/webapp/script.js

```diff
@@ -119,24 +129,31 @@ function getMessages() {
   });
 }
 
-/** Creates element for individual message. */
+// Creates element for individual message.
 function createMessage(name, text, id) {
   const container = document.createElement('div');
   const person = document.createElement('p');
   const message = document.createElement('p');
-  const remove = document.createElement('button');
   person.classList.add("name");
   message.classList.add("message");
-  remove.classList.add("remove");
 
   person.innerText = name;
   message.innerText = text;
-  remove.innerText = "X";
-  remove.onclick = () => deleteMessage(id);
 
-  container.appendChild(remove);
+  addRemoveButton(container, id)
+
   container.appendChild(person);
   container.appendChild(message);
 
   return container;
+}
+
+function addRemoveButton(container, id) {
+  if (id != 0) {
```

Comment:
This semantic isn't obvious to either the client or the server. Perhaps another way to to do this would be to rename the parameter passed to the front end "idForRemoval" or something similar.

Reply:
Fixed when implementing Objectift